source/slang/slang-ast-stmt.h

```diff
@@ -117,6 +117,17 @@ class DefaultStmt : public CaseStmtBase
     SLANG_CLASS(DefaultStmt)
 };
 
+// a `default` statement inside a `switch`
```

Comment:
The comment here is inaccurate. This AST node is unique enough that it probably deserves a decent comment.